src/modules/print/DailyPrintView.js

```diff
@@ -345,6 +357,92 @@ class DailyPrintView {
     );
   }
 
+  renderPatientInfo() {
+    const patientName = getPatientFullName(this.patient);
+    const patientBirthdate = formatBirthdate(this.patient);
+    const xOffset = this.margins.left;
+    const yOffset = this.margins.top;
+
+    this.doc
+      .lineWidth(1)
+      .fontSize(10)
+      .text(patientName, xOffset, yOffset, {
+        lineGap: 2,
+      });
+
+    const patientNameWidth = this.patientInfoBox.width = this.doc.widthOfString(patientName);
+
+    this.doc
+      .fontSize(10)
+      .text(patientBirthdate);
+
+    const patientBirthdayWidth = this.doc.widthOfString(patientBirthdate);
+    this.patientInfoBox.height = this.doc.y;
+
+    if (patientNameWidth < patientBirthdayWidth) {
+      this.patientInfoBox.width = patientBirthdayWidth;
+    }
+
+    // Render the divider between the patient info and title
+    const padding = 10;
+
+    this.doc
+      .moveTo(this.margins.left + this.patientInfoBox.width + padding, this.margins.top)
+      .lineTo(this.margins.left + this.patientInfoBox.width + padding, this.patientInfoBox.height)
+      .stroke('black');
+
+    this.dividerWidth = padding * 2 + 1;
+  }
+
+  renderTitle() {
+    const title = 'Daily View';
+    const lineHeight = this.doc.fontSize(14).currentLineHeight();
+    const xOffset = this.margins.left + this.patientInfoBox.width + 21;
+    const yOffset = (
+      this.margins.top + ((this.patientInfoBox.height - this.margins.top) / 2 - (lineHeight / 2))
+    );
+
+    this.doc.text(title, xOffset, yOffset);
+    this.titleWidth = this.doc.widthOfString(title);
+  }
+
+  renderPrintDate() {
+    const lineHeight = this.doc.fontSize(14).currentLineHeight();
+
+    // Calculate the remaining available width so we can
+    // center the print text between the patient/title text and the logo
+    const availableWidth = this.doc.page.width - _.reduce([
+      this.patientInfoBox.width,
+      this.dividerWidth,
+      this.titleWidth,
+      this.logoWidth,
+      this.margins.left,
+      this.margins.right,
+    ], (a, b) => a + b, 0);
```

Comment:
Very minor point, but does our ESLint config enforce `(a, b) => a + b` here? I'd prefer `(a, b) => (a + b)` personally, but I'm happy to stick with what the Airbnb config enforces if it enforces how you have it.